.github/ISSUE_TEMPLATE/translation_issue.yml

```diff
@@ -1,54 +1,43 @@
 name: "🌐 Localization/Translation issue"
 description: Report incorrect translations.
-labels: 
-- Issue-Bug
-- Area-Localization
-- Issue-Translation
-- Needs-Triage 
+labels: [Issue-Bug, Area-Localization, Issue-Translation, Needs-Triage]
```

Comment:
why?  vertical lists are easier to quickly parse

Reply:
https://docs.github.com/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms **always** shows them on one line. So I will follow.

Reply:
This is only visible to us, an end user doesn’t see it.  Single lines are hard to read, please revert